dev-environment.yml

```diff
@@ -29,7 +29,6 @@ dependencies:
   - pandas
   # Developer tools
   - assertpy
-  - build
```

Comment:
The conda-forge version of https://pypi.org/project/build/ is `python-build`: https://anaconda.org/conda-forge/python-build

Reply:
Thanks @imagejan! Nice catch. I fixed this here: 305d54f9afbb5960c8b96ceed73f7ff2d6274b49.